examples/tasks/async_compute.rs

```diff
@@ -0,0 +1,114 @@
+use bevy::{prelude::*, tasks::{AsyncComputeTaskPool, Task}};
+use std::time::{Instant, Duration};
+use rand::Rng;
+use futures_lite::future;
+
+/// This example shows how to use the ECS and the AsyncComputeTaskPool
+/// to spawn, poll, and complete tasks across systems and system ticks.
+
+// Number of cubes to spawn across the x, y, and z axis
+const NUM_CUBES: u32 = 6;
+
+/// This system generates tasks simulating computationally intensive
+/// work that potentially spans multiple frames/ticks. A separate
+/// system, handle_tasks, will poll the spawned tasks on subsequent
+/// frames/ticks, and use the results to spawn cubes
+fn spawn_tasks(
+    mut commands: Commands,
+    thread_pool: Res<AsyncComputeTaskPool>,
+) {
+    for x in 0 ..NUM_CUBES {
+        for y in 0 ..NUM_CUBES {
+            for z in 0 ..NUM_CUBES {
+
+                // Spawn new task on the AsyncComputeTaskPool
+                let task = thread_pool.spawn(async move {
+
+                    let mut rng = rand::thread_rng();
+                    let start_time = Instant::now();
+                    let duration = Duration::from_secs_f32(rng.gen_range(0.05..0.2));
+                    while Instant::now() - start_time < duration {
+                        // Spinning for 'duration', simulating doing hard
+                        // compute work generating translation coords!
+                    }
+
+                    // Such hard work, all done!
+                    println!("Done generating translation coords x: {} y: {} z: {}", x, y, z);

```

Comment:
Do we want to have this print statement here? While it's nice, with the default settings that 666 (216) print statement which is quite a few haha.

Reply:
it should be an `info` or `debug` rather than a `println` if we keep it

Reply:
Removed in e2c04e8